### PR TITLE
Adds use_tidy_contributing w/ check for travis

### LINKED ISSUE #143: use_tidy_contributing
Assuming it’s _possible_ that someone would want to use `use_tidy_contributing()` for a non-tidyverse package (e.g. `r-lib`), it would make more sense to build the Travis URL in the same way it’s done for `use_travis()`
```r
url <- file.path("https://travis-ci.org", gh$username, gh$repo)
```
As opposed to hardcoding it as `http://travis-ci.org/tidyverse, name` (where name is the package name, and I just skip using the `gh` package altogether. 
_But_, it would be `gh$org` in the case of tidyverse and/or r-lib, right?

Alternately, I could use `remotes` from `desc::desc_get_remotes`, but, since I'm developing off of a fork, I can't tell whether or not that would actually work 😬!

### PR DESCRIPTION
**Pull request by batpigandme**
* Fixes #143 
* Add use_tidy_contributing() (uses `gh::gh_tree_remote()` for travis link)
* Not-exported, but adds `check_uses_travis()`
* Bullet in NEWS

### CODE REVIEW

**Comment by batpigandme** [comment]
This passes `R CMD check` fine. I'm not sure how/if I turned appveyor on somehow (I've never used it before, but from the [build history](https://ci.appveyor.com/project/r-lib/usethis/history) it looks like @jennybc turned it on just a little while ago). Not sure if it's related to #146 at all, but FWICT, the `.Rproj` file causing the error (below) should be in the `.Rbuildignore` still.

`Project file "usethis.Rproj" is not a valid MSBuild project file which must be an XML.`

@hadley, not sure why this PR is on appveyor, but the previous wasn't ☹️!

**Comment by hadley** [comment]
I think @jennybc broke appveyor, so don't worry about it.

**Comment by hadley** [comment]
Thanks!

File: R/tidyverse.R [hunk 1]
```diff
@@ -110,3 +113,22 @@ use_tidy_eval <- function() {
...
+
+
+#' @export
+#' @rdname tidyverse
+#' @inheritParams use_template
+use_tidy_contributing <- function() {
+  check_uses_travis()
+
+  name <- project_name()
+  gh <- gh::gh_tree_remote(base_path)
```
Comment (jennybc) [inline R/tidyverse.R]: What is `base_path` here? It's causing a new NOTE re: `no visible binding for global variable`.

Comment (batpigandme) [reply R/tidyverse.R]: Oh, shoot— sorry, one sec.

**Pull request merged**